cli/src/commands/test-validator/index.ts

```diff
@@ -43,7 +30,7 @@ class SetupCommand extends Command {
     loader.start();
     await initTestEnv({
       skipSystemAccounts: flags.skip_system_accounts,
-      background: flags.background,
```

Comment:
why rm background flag?

Reply:
We always use background, it's enabled by default now. Do we really need blocking foreground process?

Reply:
One more reason: start-local-validator can spawn several different processes (validator, photon, etc) at the same time, so this background flag can be confusing, because all of them except validator would be in background anyway.

Reply:
yeah makes sense. previously though we actually ran it in foreground by default 
(background: Flags.boolean({
      char: "b",
      description: "Runs a test validator as a process in the background.",
      default: false,)
      
     just wondering whether the switch to default background causes the issue?

Reply:
i.e. the default command would override the default value of initTestEnv every time, unless the background flag was explicitly passed with background=true